docs/create_development_environment.md

```diff
@@ -103,7 +103,7 @@ An `OK` reply will indicating that the library was loaded successfully. Test the
 
 ## Adding Files to our Project
 
-Lets adda another file under the `src` direcotry called `test.js` that contains the following code:
+Lets add another file under the `src` direcotry called `test.js` that contains the following code:
```

Comment:
Le'ts add another file under the `src` directory called `test.js` containing the following code: